### ISSUE TITLE
how to improve accuracy ? which parameter need to tune ?

### ISSUE THREAD

**Issue by akashAD98** [chiayewken/Span-ASTE#27]
how to improve accuracy?

**Comment by chiayewken** [comment]
Hi, you can try to change bert-base to other pretrained models such as bert-large or roberta-large in the [config file](https://github.com/chiayewken/Span-ASTE/blob/main/training_config/config.jsonnet). You can also run a hyperparameter grid search for the common hyperparameters such as training epochs or learning rate.

**Comment by akashAD98** [comment]
@chiayewken thank you & do you g=have  pre-trained weight of restaurant?
<img width="460" alt="image" src="https://user-images.githubusercontent.com/62583018/185108228-8e45a15e-9011-48dc-9072-245d73d2a13c.png">

**Comment by chiayewken** [comment]
Hi, we have added the pre-trained weights restaurant data, the link is here: https://github.com/chiayewken/Span-ASTE/releases

**Comment by akashAD98** [comment]
@chiayewken thank you so much
